pygmt/gridops.py

```diff
@@ -113,3 +113,119 @@ def grdcut(grid, **kwargs):
             result = None  # if user sets an outgrid, return None
 
         return result
+
+@fmt_docstring
+@use_alias(
+           G="outgrid",
+           F="filter",
+           D="distance",
+           I="increment",
+           N="nans",
+           R="region",
+           T="toggle",
+           V="verbose",
+           f="colinfo"
+           )
+@kwargs_to_strings(R="sequence")
+
+def grdfilter(grid, **kwargs):
+    """
+    filter a grid file in the time domain using one of the selected convolution
+    or non-convolution isotropic or rectangular filters and compute distances
+    using Cartesian or Spherical geometries. The output grid file can optionally
+    be generated as a sub-region of the input (via *region*) and/or with new increment
+    (via *spacing*) or registration (via *toggle*). In this way, one may have “extra space” in
+    the input data so that the edges will not be used and the output can be within
+    one half-width of the input edges. If the filter is low-pass, then the output
+    may be less frequently sampled than the input.
+        
+    Parameters
+    ----------
+    grid : str or xarray.DataArray
+        The file name of the input grid or the grid loaded as a DataArray.
+        outgrid : str or None
+        The name of the output netCDF file with extension .nc to store the grid
+        in.
+    {F} : str
+        Name of filter type you which to apply, followed by the width
+        b: Box Car; c: Cosine Arch; g: Gaussian; o: Operator; m: Median; p: Maximum Likelihood probability; h: histogram
+        Example: 'm600' for a median filter with width of 600
```

Comment:
You can't use `{F}` in the docstring. Only things like `{R}`, `{J}`, `{U}` works, as they're common options and are defined in https://github.com/GenericMappingTools/pygmt/blob/master/pygmt/helpers/decorators.py.
```suggestion
    filter : str
        Name of filter type you which to apply, followed by the width
        b: Box Car; c: Cosine Arch; g: Gaussian; o: Operator; m: Median; p: Maximum Likelihood probability; h: histogram
        Example: 'm600' for a median filter with width of 600
```

Reply:
Ah ok.. but only in the docstring, it's ok to keep the alias F='filter'?